python/python/bystro/proteomics/tests/example_abundance_gene_MD.tsv

```diff
@@ -0,0 +1,1000 @@
+Index	NumberPSM	ProteinID	MaxPepProb	ReferenceIntensity	CPT0088900003	CPT0079270003	CPT0088920001	CPT0079300001	CPT0088550004	QC6	CPT0014450004	CPT0088570001	CPT0014470001	CPT0006730001	CPT0069190001	CPT0092730003	CPT0092740003	QC7	CPT0006630003	CPT0025920001	CPT0025880003	CPT0069160003	CPT0007870001	CPT0001270001	CPT0077110003	QC8	CPT0077140001	CPT0001260009	CPT0076350001	CPT0076330003	CPT0007860003	CPT0079380003	CPT0015810003	CPT0086030003	CPT0085670003	CPT0025230003	CPT0065750003	CPT0015730003	CPT0078800003	CPT0079230003
```

Comment:
Are these mock examples or actual datasets?

Reply:
For posterity: Cristina and I discussed this, and these are example fragpipe outputs, shared by the lab that maintains that pipeline.